main_test.go

```diff
@@ -103,9 +101,66 @@ var _ = Describe("Main", func() {
 			orgGuid   = "98C535C1-A4F8-4066-8384-733E1ADCADEE"
 		)
 
+		Context("app is renamed", func() {
+			var appsBeforeRename []cfclient.App
+			var appsAfterRename []cfclient.App
+			BeforeEach(func() {
+				appsBeforeRename = []cfclient.App{
+					{Guid: "33333333-3333-3333-3333-333333333333", Name: "foo", SpaceURL: "/v2/spaces/" + spaceGuid},
+				}
+				appsAfterRename = []cfclient.App{
+					{Guid: "33333333-3333-3333-3333-333333333333", Name: "bar", SpaceURL: "/v2/spaces/" + spaceGuid},
+				}
+
+				apiServer.AppendHandlers(
+					ghttp.CombineHandlers(
+						ghttp.VerifyRequest("GET", "/v2/apps"),
+						ghttp.RespondWithJSONEncoded(http.StatusOK, testAppResponse(appsBeforeRename)),
+					),
+					ghttp.CombineHandlers(
+						ghttp.VerifyRequest("GET", "/v2/spaces/"+spaceGuid),
+						ghttp.RespondWithJSONEncoded(http.StatusOK, testSpaceResource(spaceGuid, orgGuid)),
+					),
+					ghttp.CombineHandlers(
+						ghttp.VerifyRequest("GET", "/v2/organizations/"+orgGuid),
+						ghttp.RespondWithJSONEncoded(http.StatusOK, testOrgResource(orgGuid)),
+					),
+					ghttp.CombineHandlers(
+						ghttp.VerifyRequest("GET", "/v2/apps"),
+						ghttp.RespondWithJSONEncoded(http.StatusOK, testAppResponse(appsAfterRename)),
+					),
+					ghttp.CombineHandlers(
+						ghttp.VerifyRequest("GET", "/v2/spaces/"+spaceGuid),
+						ghttp.RespondWithJSONEncoded(http.StatusOK, testSpaceResource(spaceGuid, orgGuid)),
+					),
+					ghttp.CombineHandlers(
+						ghttp.VerifyRequest("GET", "/v2/organizations/"+orgGuid),
+						ghttp.RespondWithJSONEncoded(http.StatusOK, testOrgResource(orgGuid)),
+					),
+				)
+			})
+
+			It("update the metrics name", func() {
+				Expect(metricProc.updateApps()).To(Succeed())
+
+				var eventBeforeRename *metrics.Stream
+				Eventually(metricProc.msgChan).Should(Receive(&eventBeforeRename))
+				Expect(eventBeforeRename.App.Name).To(Equal("foo"))
+
+				checkHasNewName := func() string {
```

Comment:
`checkHasNewName` sounds a bit like a bool predicate. Could we call it `retrieveNewName` or just `newName` instead?

Reply:
🙌